### DISCUSSION TITLE
Background Blur

### DISCUSSION THREAD

**Discussion by mishnea** [ddterm/gnome-shell-extension-ddterm#465] [Ideas]
Firstly, thanks for the great extension. I think background blur would really increase readability when the background is translucent. Would this be possible to add?

**Comment by amezin** [comment]
People are reporting that "Blur my Shell" extension works fine together with ddterm now: https://github.com/ddterm/gnome-shell-extension-ddterm/issues/154#issuecomment-1972560101
